README.md

```diff
@@ -238,14 +240,16 @@ Since Chrome data suggests only ~17% of cookies use the `HttpOnly` attribute, we
 
 Note: This requirement and the requirement to only use secure protocols makes partitioned cookies behave more similarly to [HTTP State Tokens](https://github.com/mikewest/http-state-tokens).
 
-### Limit the number of cookies a third party can use in a single partition
+### Avoid a large memory footprint
 
-A third-party domain's cookie jar should have a much lower per-partition size limit than existing garbage collection thresholds ([180 cookies per domain](https://source.chromium.org/chromium/chromium/src/+/master:net/cookies/cookie_monster.h;l=104;drc=da465ccade3a693e1deac3bf01b1c83d12dbf553) in Chrome).
-We propose that third-party domains be limited to just one or some small number of cookies per-partition.
-We chose to scope the number of cookies in a single partition per third-party by domain so that a third-party could not circumvent this limit by registering new subdomains.
+One concern about introducing partitioned cookies is the proliferation of state on users' machines.
+With unpartitioned third-party cookies, a single third party only needed to set one cookie on a user's machines which could be used for cross-site requests across all top-level sites a user visits.
+After unpartitioned third-party cookies are removed, a third party will need to set one cookie per top-level context the user visits, resulting in more cookies set on user's machines.
 
-Browsers may enforce some global limit on the number of partitioned cookies in the cookie jar.
-This is to ensure that as a user visits more top-level sites over time that the number of partitioned cookies saved to their machine does not grow over time without bound.
+Browsers that wish to support partitioned cookies must impose additional limitations on the number of cookies available to a third-party domain per-partition.
+
+However, it is also necessary for user agents to design these limits in a way that does not allow malicious third parties from learning cross-site information about users.
+See [Limit the number of cookies in a top-level context's partition](#limit-the-number-of-cookies-in-a-top-level-contexts-partition) and [Applying the 180 cookies-per-domain limit](#applying-the-180-cookies-per-domain-limit) in [Alternative Solutions](#alternative-solutions) for more details.
```

Comment:
This reads as if both sections are under "Alternative solutions", but really only the latter one is.

How about:

See [Limit the number of cookies in a top-level context's partition](#limit-the-number-of-cookies-in-a-top-level-contexts-partition) for how this should be enforced. An alternative solution is also described in [Applying the 180 cookies-per-domain limit](#applying-the-180-cookies-per-domain-limit).

Reply:
Both of them are in "Alternative solutions" actually.

The one you might be thinking of is "Limit the number of cookies a third party can use in a single partition" which isn't listed here.

Reply:
Ah, sorry. LGTM then.